docs/api.md

```diff
@@ -201,6 +201,32 @@ Default: `'msg'`
 
 The string key for the 'message' in the JSON object.
 
+<a id=opt-objectkey></a>
+#### `objectKey` (String)
+
+Default: `null`
+
+If there's a chance that objects that you're logging have properties that conflict with those from pino itself (`level`, `timestamp`, `v`, `pid`, etc)
```

Comment:
to keep consistency with the docs can you use the third person objective impersonal rather than second person perspective (e.g. don't use you). See other parts of the docs for an example of description perspective

Reply:
done